.github/workflows/create_release.yml

```diff
@@ -0,0 +1,44 @@
+name: Create Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        required: true
+        type: string
+      reset_tag:
+        type: boolean
+        default: false
```

Comment:
this is a nice feature but I don't think we should ever do this vs just bumping, so maybe we can remove it here?

Reply:
Done.